samples/README.md

```diff
@@ -1,3 +1,6 @@
 # Istio Samples
 
 This directory contains sample applications highlighting various Istio features.
+
+### User Demo How-To Blogs and Examples
+* A full Google Cloud Platform based demo. Istio Ingressgateway as NEGs for HTTP(S). See [How-To: Configuring DDOS protection with Google Cloud Armor for your GKE provisioned Istio Ingressgateway service as NEGs](https://medium.com/contino-engineering/configuring-ddos-protection-with-google-cloud-armor-for-your-gke-provisioned-istio-ingressgateway-a9e862dc1683)
```

Comment:
You could add this to istio.io/docs .. We have a guides/examples section. You could add these there.

Reply:
@rshriram the istio.io/docs is not quite the same as the blog post on this topic.  The point of the Post is to engage the wider community and inspire use for the beginners, not the too technical deep-dive, at this point